setup.cfg

```diff
@@ -1,2 +1,5 @@
 [bdist_wheel]
 universal = 1
+
+[flake8]
+max-line-length = 90
```

Comment:
Any reason for 90? I think most other places we set max-line-length it's either 100 or 120.

Also put it in a .flake8 file so that Jenkins knows it needs to run flake8.

Reply:
I had it at 100 but then culled the lines in `telescope_state.py` below 90 anyway (more OCD). I'll ease it back and add .flake8. What will happen in Jenkins on flake8 errors?

Reply:
The build will fail.

Reply:
140 errors... Are we ready for this? :-P Should I add a flake8 cleanup as part of creating .flake8?

Reply:
Oh yes, this is still full of Simon-commenting, isn't it? Let's leave that for a separate PR then.

Reply:
OK, done.